docs/releases/5.0.txt

```diff
@@ -401,7 +401,8 @@ Management Commands
 Migrations
 ~~~~~~~~~~
 
-* ...
+* Support for serialization of functions decorated with :func:`functools.cache`
```

Comment:
```
`* Migrations now support serialization of functions ...
```

Reply:
That's what I had until https://github.com/django/django/pull/17256#discussion_r1324732501... I'll let you two fight it out 😂

Reply:
hahahaha I won't fight, though I prefer not to duplicate the `Migrations` bit :-)

Reply:
Ok, updating